src/schemas/json/typst.json

```diff
@@ -0,0 +1,245 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://json.schemastore.org/typst.json",
+  "title": "Typst Package Manifest",
+  "description": "Schema for typst.toml package manifest files, used by the Typst package manager and universe.",
+  "type": "object",
+  "required": ["package"],
+  "properties": {
+    "package": {
+      "type": "object",
+      "description": "Contains metadata about the package.",
+      "required": [
+        "name",
+        "version",
+        "entrypoint",
+        "authors",
+        "license",
+        "description"
+      ],
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The package's identifier in its namespace. Should use kebab-case if it contains multiple words. Should not contain 'typst'.",
+          "pattern": "^[a-z0-9][a-z0-9\\-]*$",
```

Comment:
Maybe this depends on the regex engine, but `[\\]` means you can write a backslash, it does not escape the next character. I think you wanted to use `[a-z0-9-]` for the second character group.

Reply:
Nevermind, I just realized that you had to double escape because this is JSON that will later be parsed as a Regex. Still, the backslash here is optional as far as I can tell, but having it won't allow for invalid names, so you can leave it as is.